static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -29,59 +41,99 @@ interface Props {
 interface ResourceTabSettings {
   title: string
   id: string
-  contentType: "resource" | "page"
+  contentType:
+    | typeof CONTENT_TYPE_RESOURCE
+    | typeof CONTENT_TYPE_PAGE
+    | typeof CONTENT_TYPE_COURSE_COLLECTION
+    | typeof CONTENT_TYPE_RESOURCE_COLLECTION
   resourcetype: string | null
   embeddable: boolean
+  singleColumn: boolean
+  sourceWebsiteName?: string
 }
-const RESOURCE_PICKER_TABS: ResourceTabSettings[] = [
+
+const RESOURCE_PICKER_TABS_MAIN: ResourceTabSettings[] = [
   {
     title:        "Documents",
     id:           RESOURCE_TYPE_DOCUMENT,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_DOCUMENT,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: true
   },
   {
     title:        "Videos",
     id:           RESOURCE_TYPE_VIDEO,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_VIDEO,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: false
   },
   {
     title:        "Images",
     id:           RESOURCE_TYPE_IMAGE,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_IMAGE,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: false
   },
   {
     title:        "Pages",
     id:           CONTENT_TYPE_PAGE,
     contentType:  CONTENT_TYPE_PAGE,
     resourcetype: null,
-    embeddable:   false
+    embeddable:   false,
+    singleColumn: true
   }
 ]
+const RESOURCE_PICKER_TABS_MORE: ResourceTabSettings[] = [
+  {
+    title:             "Course Collections",
+    id:                CONTENT_TYPE_COURSE_COLLECTION,
+    contentType:       CONTENT_TYPE_COURSE_COLLECTION,
+    resourcetype:      null,
+    embeddable:        false,
+    singleColumn:      true,
+    sourceWebsiteName: "ocw-www"
+  },
+  {
+    title:             "Resource Collections",
+    id:                CONTENT_TYPE_RESOURCE_COLLECTION,
+    contentType:       CONTENT_TYPE_RESOURCE_COLLECTION,
+    resourcetype:      null,
+    embeddable:        false,
+    singleColumn:      true,
+    sourceWebsiteName: "ocw-www"
```

Comment:
Hard-coding `ocw-www` here doesn't seem like the best thing to do. Do we have anything in place that says "course X belongs to "ocw-www" vs "ocw-xyz"?

One option might be to put something in course starter config as a "collection source", perhaps. I think the Instructors page does something similar as a source of instructors.

Reply:
mmm yeah, we do have something similar with the instructors, which is implemented as a 'relation' widget. The relation field takes a 'collection' prop which gives the name of the content type that it will reference, and then it also has an optional 'website' prop - if 'website' is not set, it will look in the same collection where the field is declared, and 'website' _is_ set it will look within that website. So the instructors field looks like this:

```yml
          - label: Instructors
            name: instructors
            widget: relation
            collection: instructor
            display_field: title
            multiple: true
            website: ocw-www
```

Reply:
I think it's not a bad idea to add a field to the markdown widget called `collections-source` or something along those lines, just so we can avoid hard-coding this

Reply:
I think it's fine to have this hardcoded now, and maybe we can file a tech debt issue to fix that later on (since it will require a change to the config schema and corresponding change to ocw-hugo-projects)

Reply:
Cool, I will do that next. https://github.com/mitodl/ocw-studio/issues/1052